### PR TITLE
Added method and test for well randomizer

### PR DESCRIPTION
**Pull request by LarsHalle**
Added method for well randomizer and the respective pytest

### CODE REVIEW

File: robotools/transform.py [hunk 1]
```diff
@@ -180,3 +180,64 @@ def rotate_cw(self, wells: ArrayLike) -> numpy.ndarray:
...
+        self.original_shape = original_shape
+        self.random_seed = random_seed
+        self.rng = numpy.random.RandomState(self.random_seed)
+        self.original_wells = make_well_array(*self.original_shape).flatten()
+        self.randomized_wells = self.rng.permutation(self.original_wells).tolist()
+        self.lookup = {owell: rwell for owell, rwell in zip(self.original_wells, self.randomized_wells)}
+        super().__init__()
+
+    def randomize_wells(self, wells: ArrayLike) -> numpy.ndarray:
+        """Randomize the given wells with in init given random state and assignment.
```
Comment (Y0dler) [inline robotools/transform.py:207]:
```suggestion
        """Randomize the given wells with the random state and assignment specified in __init__.
```

File: robotools/transform.py [hunk 2]
```diff
@@ -180,3 +180,64 @@ def rotate_cw(self, wells: ArrayLike) -> numpy.ndarray:
...
+            Array of well ids
+        """
+
+        self.input_wells = numpy.array(wells)
+        self.randomized_output_wells = [self.lookup.get(well) for well in self.input_wells]
+
+        return numpy.array(self.randomized_output_wells)
+
+    def derandomize_wells(self, wells: ArrayLike) -> numpy.ndarray:
+        """Derandomize the given wells with in init given random state and assignment.
```
Comment (Y0dler) [inline robotools/transform.py:226]:
```suggestion
        """Derandomize the given wells with the random state and assignment specified in __init__.
```

File: robotools/transform.py [hunk 3]
```diff
@@ -180,3 +180,64 @@ def rotate_cw(self, wells: ArrayLike) -> numpy.ndarray:
...
+    """Helper object to rendomize a set of well IDs within a MTP."""
+
+    def __init__(self, original_shape: tuple, random_seed: int) -> None:
+        """Create a helper object for randomizing wells.
+
+        Parameters
+        ----------
+        original_shape : tuple
+            (n_rows, n_cols) of all wells in the source labware
+        random_state_int : int
```
Comment (Y0dler) [inline robotools/transform.py:195]:
```suggestion
        random_seed : int
```

**Review by Y0dler — CHANGES_REQUESTED** [review]
I understood the general structure and it seems correct. I just have a few comments regarding langauge/documentation.

**Comment by LarsHalle** [comment]
Thank you @Y0dler !

**Review by Y0dler — APPROVED** [review]

**Pull request merged**